lib/dialects/postgres/query-generator.js

```diff
@@ -375,7 +375,7 @@ var QueryGenerator = {
 
     var replacements = {
       table: this.quoteIdentifiers(tableName),
-      where: this.getWhereConditions(where),
+      where: this.getWhereConditions(where, null, model, options),
```

Comment:
`getWhereConditions` is deprecated, perhaps change it so it calls the new where generator directly.